xarray/namedarray/core.py

```diff
@@ -340,15 +555,15 @@ def chunks(self) -> tuple[tuple[int, ...], ...] | None:
         xarray.unify_chunks
         """
         data = self._data
-        if is_chunked_duck_array(data):
+        if isinstance(data, _chunkedarray):
```

Comment:
A more lightweight (and faster if that matters at all) version would be

```suggestion
        return getattr(self.data, "chunks", None)
```

Reply:
Yes, I was thinking this was more readable though. You didn't like the `is_chunked_duck_array` style in an earlier PR. :D